src/praisonai/praisonai/kanban/sqlite_store.py

```
@@ -0,0 +1,513 @@
+                return self.get_task(task_id)
+            
+            update_fields.append("updated_at = ?")
+            update_fields.append("version = ?")
+            values.extend([now.isoformat(), new_version, task_id, current_version])
+            
+            # CAS update with version check
+            result = conn.execute(f"""
+                UPDATE tasks 
+                SET {', '.join(update_fields)}
+                WHERE id = ? AND version = ?
+            """, values)
```

Ruff (0.15.14)

[error] Possible SQL injection vector through string-based query construction

(S608)